chart/skywalking/templates/oap-deployment.yaml

```diff
@@ -142,3 +142,17 @@ spec:
         - name: {{ $key }}
           value: {{ $value | quote }}
         {{- end }}
+
+        volumeMounts:
+          {{- if (.Files.Glob "files/conf.d/oap/application.yml") }}
+          - name: skywalking-override-application-yml
+            mountPath: /skywalking/ext-config/application.yml
```

Comment:
we can mount to `/skywalking/config` directly instead of ext folder, which is used for overriding a batch of files.

Reply:
Could refer this? https://github.com/apache/skywalking/blob/master/docker/oap/docker-entrypoint.sh#L26-L28.
There is a file copy mechanism in the codes for a long time, just not documented.

Reply:
Done.

> Could refer this? https://github.com/apache/skywalking/blob/master/docker/oap/docker-entrypoint.sh#L26-L28.

That's exactly where I referred to